src/engraving/libmscore/cmd.cpp

```diff
@@ -728,6 +731,23 @@ void Score::addInterval(int val, const std::vector<Note*>& nl)
 
         select(note, SelectType::SINGLE, 0);
     }
+    unsigned i = 0;
+    for (Note* on : nl) {
+        if (on->tieFor()) {
+            auto it = find(nl.begin(), nl.end(), on->tieFor()->endNote());
+            if (it != nl.end()) {
+                unsigned index = it - nl.begin();
+                Tie* tie = Factory::createTie(this->dummy());
+                tie->setStartNote(notes[i]);
+                tie->setEndNote(notes[index]);
+                tie->setTrack(notes[i]->track());
+                tie->setTick(notes[i]->chord()->segment()->tick());
+                tie->setTicks(notes[index]->chord()->segment()->tick() - notes[i]->chord()->segment()->tick());
```

Comment:
I believe these lines can be done slightly simpler:
```suggestion
                tie->setTick(notes[i]->tick());
                tie->setTick2(notes[index]->tick());
```